### PR TITLE
test: added more tests

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/items/__tests__/item.service.spec.ts [hunk 1]
```diff
@@ -256,5 +256,37 @@ describe('ItemService', () => {
         expect(actual).toEqual(expected);
       });
     });
+    describe('and the execution is unsuccessful', () => {
+      it('throw a NotFoundException when item is not found', async () => {
```
Comment (mmiranda-stackit) [inline src/items/__tests__/item.service.spec.ts:260]: try to make the reading easier. When you put the it clause, you put 'should' first and then describe what should happen ;)

File: src/items/__tests__/item.service.spec.ts [hunk 2]
```diff
@@ -256,5 +256,37 @@ describe('ItemService', () => {
...
+
+        const exception = () =>
+          service.delistAnItem({ itemId: unexistingItemId, address: unexistingAddress });
+
+        await expect(exception).rejects.toThrow(NotFoundException);
+        await expect(exception).rejects.toEqual(new NotFoundException(errorMessage));
+
+        expect(itemRepository.delistAnItem).not.toHaveBeenCalled();
+      });
+      it('throw a NotFoundException when account is not found', async () => {
```
Comment (mmiranda-stackit) [inline src/items/__tests__/item.service.spec.ts:275]: You have two its inside the same describe scenario. Normally you have one it per scenario

**Review by mmiranda-stackit — COMMENTED** [review]
see my comments

**Review by juliswer — APPROVED** [review]
Looks good

**Pull request merged**